src/portalbackend/PortalBackend.Migrations/Migrations/20230706131002_CPLP-2841-CompanyLinkedServiceAccountsView.cs

```diff
@@ -0,0 +1,63 @@
+/********************************************************************************
+ * Copyright (c) 2021, 2023 BMW Group AG
+ * Copyright (c) 2021, 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Org.Eclipse.TractusX.Portal.Backend.PortalBackend.Migrations.Migrations
+{
+    public partial class CPLP2841CompanyLinkedServiceAccountsView : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"CREATE OR REPLACE VIEW CompanyLinkedServiceAccounts AS
```

Comment:
Since we're using schemas we should use them for the views as well to have them at the same schema as all data
```suggestion
            migrationBuilder.Sql(@"CREATE OR REPLACE VIEW portal.CompanyLinkedServiceAccounts AS
```

Reply:
fixed